pydevices/acq400Devices/acq400_base.py

```diff
@@ -23,26 +23,28 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 
-import sys
 import threading
 import socket
 import time
-import inspect
-
-import numpy as np
+from queue import Queue, Empty
+import numpy
 import MDSplus
 
-if MDSplus.version.ispy3:
-    from queue import Queue, Empty
-else:
-    from Queue import Queue, Empty
+try:
+    acq400_hapi = __import__('acq400_hapi', globals(), level=1)
```

Comment:
dont import third party packages on module level
https://www.mdsplus.org/index.php?title=pydevice_dev

Reply:
Yes, you are correct about this.